lib/galaxy/config/__init__.py

```diff
@@ -817,6 +817,7 @@ def _process_config(self, kwargs):
         self.interactivetools_map = self._in_root_dir(kwargs.get("interactivetools_map", self._in_data_dir("interactivetools_map.sqlite")))
         self.interactivetools_prefix = kwargs.get("interactivetools_prefix", "interactivetool")
         self.interactivetools_proxy_host = kwargs.get("interactivetools_proxy_host", None)
+        self.interactivetools_use_k8s_ingress = kwargs.get("interactivetools_use_k8s_ingress", False)
```

Comment:
Could you, please, add this to the schema instead? Similar to `interactivetools_proxy_host`?

Reply:
I can. Was wondering if this should just be a runner param instead though, given that it really only works wit the k8s runner. I'll look into how easy it is to change it to that

Reply:
I think that would make sense. I guess it's hypothetical, but in theory deployers might want to run have more than one runner type that can run interactive tools, so that's another reason to put this in the runner parameters.

Reply:
Done in https://github.com/galaxyproject/galaxy/pull/11383/commits/63510bea237eb1ef58aab8294e06e65fb3b743f3 (left default as if not added so in k8s destination have to specify: `require_monitor: False` but nothing will break)